test/support/fixtures/dynamic/interface_cycle_schema.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Absinthe.TestSupport.Schema.InterfaceCycleSchema do
+  use Absinthe.Schema
+
+  @sdl """
+  schema {
+    query: Query
+  }
+
+  type Query {
+    node: Node
+  }
+
+  interface Node implements Named & Node {
+    id: ID!
+    name: String
+  }
+
+  interface Named implements Node & Named {
+    id: ID!
+    name: String
+  }
+  """
+
+  import_sdl @sdl
+  # query do
```

Comment:
Leftover lines?

Reply:
yes, let me fix that

Reply:
https://github.com/absinthe-graphql/absinthe/pull/1013